Harvest.Web/Startup.cs

```diff
@@ -93,6 +93,23 @@ public void ConfigureServices(IServiceCollection services)
                         return;
                     }
 
+                    // Sometimes CAS doesn't return the required IAM ID
+                    // If this happens, we take the reliable Kerberos (NameIdentifier claim) and use it to lookup IAM ID
+                    if (!identity.HasClaim(c => c.Type == "ucdPersonIAMID"))
+                    {
+                        var identityService = context.HttpContext.RequestServices.GetRequiredService<IIdentityService>();

```

Comment:
Looks like CAS stage is fixed, but this is probably good to keep.
Throw in a log error message to let un know?